tutorminio/patches/k8s-deployments

```diff
@@ -19,7 +19,11 @@ spec:
       containers:
         - name: minio
           image: {{ DOCKER_REGISTRY }}{{ MINIO_DOCKER_IMAGE }}
+          {% if MINIO_MODE == "server" %}
           args: ["server", "--address", ":9000", "/data"]
+          {% else %}
+          args: ["gateway", "{{ MINIO_GATEWAY }}", "{{ MINIO_GATEWAY_ENDPOINT }}", "--address", ":9000"]
```

Comment:
Don't you need to provide specific credentials, depending on the gateway type? For instance, Azure supports the MINIO_AZURE_CHUNK_SIZE_MB setting: https://docs.minio.io/docs/minio-gateway-for-azure.html